docs/learn/learn-proxies.md

```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
+:::
 
-  ![anon proxy info](../assets/kill-proxy-1.png)
+Anonymous proxies are very different from other proxy types. Proxies that we described so far are _existing accounts_ assigned as proxies by a primary account. These proxies act on behalf of the primary account reducing the exposure of the primary account's private key. Remember, the more often we use an account's private key to sign for different things, the more we expose that key to the internet, increasing the visibility of that account. The purpose of a proxy is thus to draw the attention of potential attackers away from the primary account, as proxies' private keys will be used most of the time to perform actions on behalf of the primary account. We can imagine proxies as bodyguards of a VIP, loyal and ready to risk their lives to ensure the VIP's protection.
 
-- **Step 1**: Go to https://polkadot.js.org/apps/#/accounts (make sure you are on correct network).
-- **Step 2**: Click `Proxied` and add your address, name it `ANON PROXY`. You should now see this address
-  in accounts. Now you need to call `killAnonymous` from the anonymous proxy. It is important to note that anonymous proxies *work backwards*; the original account acts as the proxy.
+![anonymous proxies](../assets/proxy_vs_anon.png)
 
-  ![add proxy to delete](../assets/kill-proxy-2.png)
+Anonymous proxies are new accounts that are _created_ (not assigned) by a primary account. That primary account then acts as `any proxy` on behalf of the anonymous proxy. Note that, to avoid confusion, in this section we will refer to `Anon` for anonymous proxy. Anon are **key-less anonymous accounts** as they do have an address but do not have a corresponding private key and, in some sense, are anonymous as nobody has a private key to control it. The use of the Anon is strictly bound to the relationship between the Anon and the `Any` proxy (not necessarily the one who created the Anon in the first place). Once that relationship is broken, the Anon will be inaccessible (even if visible on the Polkadot-JS UI). Also, Anon accounts are non-deterministic, this means that if we lose one Anon, the next one we create from the same primary account will have a different address.
 
-- **Step 3**: Go to https://polkadot.js.org/apps/#/extrinsics
-- **Step 4**: Call extrinsic `proxy.killAnonymous` using the selected account ANON PROXY and the following parameters:
+Anon accounts cannot sign anything because they do not have private keys. As a consequence, Anon cannot act as proxies (which is admittedly confusing given their original name), and cannot be included in multi-sig. An Anon account acting as a proxy will not be able to sign anything on behalf of the primary account, and an Anon within a multi-sig will be able neither to initiate nor to approve any call.
```

Comment:
Removing factually incorrect content

```suggestion
`Anon` accounts cannot sign anything because they do not have private keys. As a consequence, `Anon` cannot act as proxies (which is admittedly confusing given their original name). An `Anon` account acting as a proxy will not be able to sign anything on behalf of the primary account.
```

Reply:
Right! I wrote this before discovering proxy calls.